pkg/diagnose/connectivity/hooks.go

```diff
@@ -51,29 +53,36 @@ var (
 	tlsColorFunc = color.YellowString
 )
 
+// writeWrapper is used to pass a io.Writer variable to the hooks so that they
+// can output into a terminal if the command is used via CLI or into a buffer
+// when requesting a flare
+type writerWrapper struct {
```

Comment:
I think the name can be improved to better reflect what struct does, ie implements hooks for `ClientTrace`. You can name it `connectivityHooks` for example.

Reply:
yeah you're right. Done here 14970e4d79ac03d8b2826fc6bf5a3579c9c0c157